nima/observe/metrics/src/main/java/io/helidon/nima/observe/metrics/MetricsFeature.java

```diff
@@ -181,6 +181,11 @@ public void afterStop() {
         }
     }
 
+    @Override
+    protected void context(String context) {
+        super.context(context);
```

Comment:
Is this left-over? this override doesn't do anything.

Reply:
Except make the method visible within package to `MetricObserveProvider` (same as with health).

Reply:
roger that.